src/components/molecules/OakFilterDrawer/OakFilterDrawer.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { StoryObj, Meta } from "@storybook/react";
+import { useArgs } from "@storybook/preview-api";
+
+import { OakFilterDrawer } from "./OakFilterDrawer";
+
+import { OakSecondaryButton } from "@/components/molecules/OakSecondaryButton";
+import { OakBox, OakFlex, OakHeading } from "@/components/atoms";
+import { OakSearchFilterCheckBox } from "@/components/organisms";
+import { OakRadioGroup } from "@/components/molecules/OakRadioGroup";
+import { OakPrimaryButton } from "@/components/molecules/OakPrimaryButton";
+import { OakRadioButton } from "@/components/molecules/OakRadioButton";
+
+const meta: Meta<typeof OakFilterDrawer> = {
+  component: OakFilterDrawer,
+  tags: ["autodocs"],
+  title: "components/molecules/OakFilterDrawer",
+  parameters: {
+    controls: {
+      include: ["isOpen", "onClose", "clearAllInputs"],
+    },
+  },
+  args: {
+    "aria-label": "Example filter drawer",
+    "aria-description": "This is an example filter drawer",
+  },
+  render: function Render(args) {
+    const [, updateArgs] = useArgs();
+    const onClose = () => updateArgs({ isOpen: false });
+    const onOpen = () => updateArgs({ isOpen: true });
+
+    return (
+      <>
+        <OakSecondaryButton onClick={onOpen}>
+          Open filter drawer
+        </OakSecondaryButton>
+        <OakFilterDrawer
+          {...args}
+          onClose={onClose}
+          footerSlot={
+            <OakPrimaryButton onClick={onClose}>
+              Show results (23)
```

Comment:
Will the number in the brackets update based on the filters selected?

Reply:
Yeah this is just being hard coded in the storybook file as there aren't any units to filter through, but will be dynamic when plugged in